sunpy/cm/cm.py

```diff
@@ -156,7 +166,12 @@ def show_colormaps():
     ----------
 
     """
-    maps = sorted(cmlist)
+
+    if mission:
```

Comment:
not sure this needed, because `if mission` and `if missions is not None` do almost the same thing, apart from not allowing through empty lists and False